fission/src/ui/panels/GraphicsSettingsPanel.tsx

```diff
@@ -0,0 +1,184 @@
+                                World.SceneRenderer.changeCSMSettings({
+                                    cascades: value as number,
+
+                                    maxFar: maxFar,
+                                    lightIntensity: lightIntensity,
+                                    fancyShadows: fancyShadows,
+                                    shadowMapSize: shadowMapSize,
+                                    antiAliasing: antiAliasing,
+                                })
+                            }}
+                            step={1}
+                        />
+                        <Slider
+                            min={MIN_SHADOW_MAP_SIZE}
+                            max={World.SceneRenderer.renderer.capabilities.maxTextureSize}
+                            value={shadowMapSize}
+                            label="Shadow Map Size"
+                            onChange={(_, value: number | number[]) => {
+                                setShadowMapSize(value as number)
+                                World.SceneRenderer.changeCSMSettings({
+                                    shadowMapSize: value as number,
+                                    maxFar: maxFar,
+                                    lightIntensity: lightIntensity,
+                                    fancyShadows: fancyShadows,
+                                    cascades: cascades,
+                                    antiAliasing: antiAliasing,
+                                })
+                            }}
+                            step={1024}
+                        />
+                        {Spacer(10)}
+                        <Box alignSelf={"center"}>
+                            <Button
+                                value="Reset Default"
+                                onClick={() => {
+                                    setShadowMapSize(4096)
+                                    setMaxFar(30)
+                                    setLightIntensity(5)
+                                    setCascades(4)
+
+                                    World.SceneRenderer.changeCSMSettings({
+                                        shadowMapSize: 4096,
+                                        maxFar: 30,
+                                        lightIntensity: 5,
+                                        fancyShadows: fancyShadows,
+                                        cascades: 4,
+                                        antiAliasing: antiAliasing,
+                                    })
+                                }}
+                            />
+                        </Box>
+                    </>
+                ) : (
+                    <></>
+                )}
+                <div className="flex items-center justify-center mt-1 mb-0.5 mx-[5%]">
+                    <SectionLabel size={LabelSize.Medium} className="text-center">
+                        Hot-reload Settings
+                    </SectionLabel>
```

Comment:
I'm partial to the opinion that this label should be removed. At first I thought it was some sort of button or toggle and it seemed out of place.